docs/realtime/guide.md

```diff
@@ -0,0 +1,143 @@
+
+Key differences from regular agents:
+
+-   Model choice is configured at the session level, not the agent level.
+-   No structured output support (`outputType` is not supported).
+-   Voice can be configured per agent but cannot be changed after the first agent speaks.
+-   All other features like tools, handoffs, and instructions work the same way.
+
+## Session configuration
+
+### Model settings
+
+The session configuration allows you to control the underlying realtime model behavior. You can configure the model name (such as `gpt-4o-realtime-preview`), voice selection (alloy, echo, fable, onyx, nova, shimmer), and supported modalities (text and/or audio). Audio formats can be set for both input and output, with PCM16 being the default.
+
+### Audio configuration
+
+Audio settings control how the session handles voice input and output. You can configure input audio transcription using models like Whisper, set language preferences, and provide transcription prompts to improve accuracy for domain-specific terms. Turn detection settings control when the agent should start and stop responding, with options for voice activity detection thresholds, silence duration, and padding around detected speech.
+
+## Tools and Functions
+
+### Adding Tools
+
+Just like regular agents, realtime agents support function tools that execute during conversations:
+
+```python
+from agents import function_tool
+
+@function_tool
+def get_weather(city: str) -> str:
+    """Get current weather for a city."""
+    # Your weather API logic here
+    return f"The weather in {city} is sunny, 72°F"
+
+@function_tool
+def book_appointment(date: str, time: str, service: str) -> str:
+    """Book an appointment."""
+    # Your booking logic here
+    return f"Appointment booked for {service} on {date} at {time}"
+
+agent = RealtimeAgent(
+    name="Assistant",
+    instructions="You can help with weather and appointments.",
+    tools=[get_weather, book_appointment],
+)
+```
+
+## Handoffs
+
+### Creating Handoffs
+
+Handoffs allow transferring conversations between specialized agents.
+
+```python
+from agents.realtime import realtime_handoff
+
+# Specialized agents
+billing_agent = RealtimeAgent(
+    name="Billing Support",
+    instructions="You specialize in billing and payment issues.",
+)
+
+technical_agent = RealtimeAgent(
+    name="Technical Support",
+    instructions="You handle technical troubleshooting.",
+)
+
+# Main agent with handoffs
+main_agent = RealtimeAgent(
+    name="Customer Service",
+    instructions="You are the main customer service agent. Hand off to specialists when needed.",
+    handoffs=[
+        realtime_handoff(billing_agent, tool_description="Transfer to billing support"),
+        realtime_handoff(technical_agent, tool_description="Transfer to technical support"),
+    ]
```

Comment:
What is the difference between just putting in the agent and using "realtime_handoff"? In your web demo you just have "handoffs=[agent]"?